visidata/plugins.py

```diff
@@ -54,37 +63,49 @@ def installPlugin(self, plugin):
 
     @asyncthread
     def _install(self, plugin):
-        outpath = _plugin_path(plugin)
-        os.makedirs(outpath.parent, exist_ok=True)
+        exists = False
+        with Path(_plugin_init()).open_text(mode='r') as fprc:
+            r = re.compile(r'^{}\W'.format(_plugin_import(plugin)))
+            for line in fprc.readlines():
+                if r.match(line):
+                    exists = True
+                    warning("plugin already loaded")
 
-        with urlcache(plugin.url, 0).open_text() as pyfp:
-            with outpath.open_text(mode='w') as outfp:
-                outfp.write(pyfp.read())
+        if not exists:
+            outpath = _plugin_path(plugin)
 
-        if plugin.requirements:
-            p = subprocess.Popen(['pip3', 'install']+plugin.requirements.split())
```

Comment:
It looks like requirements won't be installed anymore; will the user need to install them manually now?